examples/index.json

```
@@ -1894,6 +1894,15 @@
                 "disablePlayground": true,
                 "isLearnByExample": false
             },
+            {
+                "name": "Bidirectional streaming RPC copy",
+                "url": "grpc-bidirectional-streaming-copy",
```

```suggestion
                "url": "grpc-bidirectional-streaming-copy",
```
Is `copy` a intended word?

---

Yes. Added it to keep both directories to compare the changes easily. Will delete it later.